From: Jon Ludlam <jonathan.ludlam@eu.citrix.com>
Date: Thu, 8 Dec 2011 17:06:01 +0000
Subject: remove-xencert

---
 Makefile |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/Makefile b/Makefile
index fbe0213..1c15229 100644
--- a/Makefile
+++ b/Makefile
@@ -122,7 +122,6 @@ install:
 	install -m 755 drivers/runvmpr $(SM_STAGING)$(SM_DEST)
 	install -m 755 drivers/vmpr $(SM_STAGING)$(PLUGIN_SCRIPT_DEST)
 	ln -sf $(SM_DEST)lcache.py $(SM_STAGING)$(BIN_DEST)tapdisk-cache-stats
-	cp -rf XenCert $(SM_STAGING)$(DEBUG_DEST)
 
 .PHONY: clean
 clean:
-- 
