From: Jon Ludlam <jonathan.ludlam@eu.citrix.com>
Date: Thu, 8 Dec 2011 17:06:02 +0000
Subject: fix-makefile

---
 Makefile            |   15 ++++++++-------
 snapwatchd/Makefile |   12 ++++++------
 2 files changed, 14 insertions(+), 13 deletions(-)

diff --git a/Makefile b/Makefile
index a1ff8c5..fbfbbf5 100644
--- a/Makefile
+++ b/Makefile
@@ -67,9 +67,10 @@ SM_STAGING := $(DESTDIR)
 SM_STAMP := $(MY_OBJ_DIR)/.staging_stamp
 
 .PHONY: build
+build:
 	make -C dcopy 
 	make -C snapwatchd
-	make -C mpathroot
+	#make -C mpathroot
 
 .PHONY: install
 install: 
@@ -99,7 +100,7 @@ install:
 	cd $(SM_STAGING)$(SM_DEST) && rm -f LVHDSR && ln -sf LVHDSR.py LVMSR
 	cd $(SM_STAGING)$(SM_DEST) && rm -f LVHDoISCSISR && ln -sf LVHDoISCSISR.py LVMoISCSISR
 	cd $(SM_STAGING)$(SM_DEST) && rm -f LVHDoHBASR && ln -sf LVHDoHBASR.py LVMoHBASR
-	ln -sf $(SM_DEST)mpathutil.py $(SM_STAGING)/sbin/mpathutil
+	#ln -sf $(SM_DEST)mpathutil.py $(SM_STAGING)/sbin/mpathutil
 	install -m 755 drivers/02-vhdcleanup $(SM_STAGING)$(MASTER_SCRIPT_DEST)
 	install -m 755 drivers/lvhd-thin $(SM_STAGING)$(PLUGIN_SCRIPT_DEST)
 	install -m 755 drivers/on-slave $(SM_STAGING)$(PLUGIN_SCRIPT_DEST)
@@ -115,12 +116,12 @@ install:
 	install -m 755 scripts/check-device-sharing $(SM_STAGING)$(LIBEXEC)
 	$(MAKE) -C dcopy install DESTDIR=$(SM_STAGING)
 	$(MAKE) -C snapwatchd install DESTDIR=$(SM_STAGING)
-	$(MAKE) -C mpathroot install DESTDIR=$(SM_STAGING)
+#	$(MAKE) -C mpathroot install DESTDIR=$(SM_STAGING)
 	ln -sf $(SM_DEST)blktap2.py $(SM_STAGING)$(BIN_DEST)/blktap2
-	install -m 755 -d $(SM_STAGING)$(CRON_DEST)
-	install -m 644 $(CRON_JOBS:%=etc/cron.d/%) -t $(SM_STAGING)$(CRON_DEST)
-	install -m 755 drivers/runvmpr $(SM_STAGING)$(SM_DEST)
-	install -m 755 drivers/vmpr $(SM_STAGING)$(PLUGIN_SCRIPT_DEST)
+#	install -m 755 -d $(SM_STAGING)$(CRON_DEST)
+#	install -m 644 $(CRON_JOBS:%=etc/cron.d/%) -t $(SM_STAGING)$(CRON_DEST)
+#	install -m 755 drivers/runvmpr $(SM_STAGING)$(SM_DEST)
+#	install -m 755 drivers/vmpr $(SM_STAGING)$(PLUGIN_SCRIPT_DEST)
 	ln -sf $(SM_DEST)lcache.py $(SM_STAGING)$(BIN_DEST)tapdisk-cache-stats
 
 .PHONY: clean
diff --git a/snapwatchd/Makefile b/snapwatchd/Makefile
index addcbfb..4cdefe2 100644
--- a/snapwatchd/Makefile
+++ b/snapwatchd/Makefile
@@ -5,7 +5,7 @@ PREFIX ?= /usr/lib/xcp/sm/snapwatchd
 INITDIR ?= /etc/rc.d/init.d
 DESTDIR ?= 
 
-PYTHON_INCLUDE ?= /usr/include/python2.4
+PYTHON_INCLUDE ?= /usr/include/python2.7
 
 CFLAGS ?= -O2
 CFLAGS += -I$(PYTHON_INCLUDE) -I$(XS_INCLUDE) -fPIC
@@ -17,13 +17,13 @@ all: _xslib.so
 	@ :
 
 .PHONY: install
-install: _xslib.so xslib.py snapwatchd
+install: _xslib.so xslib.py
 	mkdir -p $(DESTDIR)$(PREFIX)
 	install -m 644 $^ $(DESTDIR)$(PREFIX)
-	chmod +x $(DESTDIR)$(PREFIX)/snapwatchd
-	mkdir -p $(DESTDIR)$(INITDIR)
-	cp snapwatchd.init $(DESTDIR)$(INITDIR)/snapwatchd
-	chmod +x $(DESTDIR)$(INITDIR)/snapwatchd
+	#chmod +x $(DESTDIR)$(PREFIX)/snapwatchd
+	#mkdir -p $(DESTDIR)$(INITDIR)
+	#cp snapwatchd.init $(DESTDIR)$(INITDIR)/snapwatchd
+	#chmod +x $(DESTDIR)$(INITDIR)/snapwatchd
 
 %.py %_wrap.c: %.i
 	$(SWIG) -python $<
-- 
